applications/nrf5340_audio/README.rst

```diff
@@ -1288,8 +1293,7 @@ Testing the BIS mode is identical to `Testing the default CIS mode`_, except for
 * Pressing the **PLAY/PAUSE** button on the gateway will respectively start or stop the stream for all headsets listening in.
 * Pressing the **BTN 4** button on a headset will change the active audio stream.
   The default configuration of the BIS mode supports two audio streams (left and right).
-* Pressing the **BTN 5** button on a headset will change the gateway source for the audio stream.
-  The default configuration of the BIS mode supports two gateways (``CONFIG_BT_DEVICE_NAME`` and ``CONFIG_BT_DEVICE_NAME_ALT``).
+* Pressing the **BTN 5** button on a headset will change the gateway source for the audio stream (after `Enabling the BIS mode with two gateways`_).
```

Comment:
```suggestion
* Pressing the **BTN 5** button on a headset will change the gateway source for the audio stream (after `Enabling the BIS mode with two gateways`_). If a second gateway is not present the headset will not play audio.
```

When BTN 5 is pressed the headset will try to scan for the second gateway even if it is not there (even if we have not done the steps in `Enabling the BIS mode with two gateways`).